buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json

```diff
@@ -6,14 +6,14 @@
       "alwaysRun": false,
       "displayName": "Install Signing Plugin",
       "timeoutInMinutes": 0,
-      "condition": "ne(variables['PB_SignType'], 'oss')",
+      "condition": "and(succeeded(), or(eq(variables['PB_SignType'], 'real'), eq(variables['PB_SignType'], 'test'))) ",
```

Comment:
A more compact way to express this is `and(succeeded(), in(variables.PB_SignType, 'real', 'test'))`